docs/core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments.mdx

```diff
@@ -2,8 +2,58 @@
 sidebar_label: 'Getting Started With Bot Sequences and Segments'
 ---
 
+import SequenceCard from './img/sequence_card.png';
+import SequenceDeleteDialog from './img/sequence_delete.png';
+import CreateSequenceButton from './img/create_sequence_button.png';
+import ReloadIcon from './img/reload.png';
+
 # Getting Started With Bot Sequences and Segments
 
-import Placeholder from '../../partials/_placeholder.mdx';
+A **Bot Sequence** is a series of automated actions that can be used to assert certain behaviours or outcomes within your game. Sequences are composed of individual actions, called Segments, which are executed in order.
+
+![Bot Sequence List in RGOverlayCanvas](img/sequences_list.png)
+
+## Starting A Bot Sequence
+
+After [adding the RGOverlayCanvas prefab to your scene](../the-in-game-overlay.mdx), click on the small Williward face icon to open the overlay. With the overlay opened, you can see the list of Bot Sequences that are available to you. Each Sequence is represented by a card, with three possible actions:
```

Comment:
mention something about the icon being in the toolbar, so the user can easily locate it.

also I don't think we need that "in-game overlay" section anymore if we're including how to add it in the getting-started section and go over its functionality here. I'd link to that installation section instead

```suggestion
After [adding the RGOverlayCanvas prefab to your scene](../the-in-game-overlay.mdx), click on the Williward icon in the toolbar to open the overlay. With the overlay opened, you can see the list of Bot Sequences that are available to you. Each Sequence is represented by a card, with three possible actions:
```

Reply:
Maybe we can specifically name this the "RG Toolbar" both here and in the installation guide?

Reply:
@abeizer good call on the wording. I've updated that and also the link to the installation section

Reply:
@vontell RG Toolbar it is!